Makefile

```diff
@@ -1,16 +1,22 @@
-CLD_HOST   ?= y.cld.gov.au
-DEPLOY_ENV ?= notifications
-CF_API     ?= https://api.system.$(CLD_HOST)
-CF_ORG     ?= dta
-CF_SPACE   ?= $(DEPLOY_ENV)
-CF_HOME    ?= $(HOME)
+CLD_HOST    ?= y.cld.gov.au
+DEPLOY_ENV  ?= notifications
+CF_API      ?= https://api.system.$(CLD_HOST)
+CF_ORG      ?= dta
+CF_SPACE    ?= $(DEPLOY_ENV)
```

Comment:
@dagoof you can remove `DEPLOY_ENV` and just make `CF_SPACE= notifications`. Unless you were planning on using `DEPLOY_ENV` for other things later?

Reply:
I think I wanted to decouple our notion of environment /app name from the deploy mechanism. but I don't know if it even matters anymore